debian/control

```diff
@@ -0,0 +1,20 @@
+Source: matrix-synapse
+Section: contrib/python
+Priority: extra
+Maintainer: Synapse Developers <synapse@matrix.org>
+Build-Depends: debhelper (>= 9), python3, python3-dev, libffi-dev, libssl-dev, libjpeg-dev, libxslt1-dev, dh-virtualenv (>= 0.10), tar, lsb-release
```

Comment:
please can you break up the dependencies lines into one package per line, sorted alphabetically